apps/cnquery/cmd/root.go

```diff
@@ -132,17 +132,20 @@ func sysInfoHeader(sysInfo *sysinfo.SystemInfo, features cnquery.Features) range
 	)
 
 	h := http.Header{}
-	h.Set(HttpHeaderUserAgent, scope.XInfoHeader(map[string]string{
+	info := map[string]string{
 		"cnquery": cnquery.Version,
 		"build":   cnquery.Build,
-		"PN":      sysInfo.Platform.Name,
-		"PR":      sysInfo.Platform.Version,
-		"PA":      sysInfo.Platform.Arch,
-		"IP":      sysInfo.IP,
-		"HN":      sysInfo.Hostname,
-	}))
+	}
+	if sysInfo != nil {
```

Comment:
the only real change here is this nil checking, which i did to avoid setting a log.Fatal() if we can't fetch the sys info